src/custom-elements/src/components/expansion-panel/expansion-panel.scss

```diff
@@ -1,11 +1,8 @@
-@import 'components/icons/webfont-font-face';
 :host(chi-expansion-panel) {
     @import '../../global/styles/common';
-    @import 'components/expansion-panel/expansion-panel';
-
     &:not(:last-child) {
         .chi-epanel:not(.-bordered) {
-            border-bottom: 0.0625rem solid $color-border-base;
+            border-bottom: 0.0625rem solid #DADEE2;
         }
     }
```

Comment:
Can you remove this as it's not themable and place the following on line 256 in /src/chi/components/expansion-panel/expansion-panel.scss

Note: I've changed **last-child** to **last-of-type**

```
chi-expansion-panel {
  &:not(:last-of-type) {
    .chi-epanel:not(.-bordered) {
      border-bottom: 0.0625rem solid $color-border-base;
    }
  }
}
```